### ISSUE TITLE
⌨ Page load is not being announced

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#13396]
### ⌨ Accessibility and Expected Behaviour

When selecting any of the links under "your information" on the applicant dashboard VoiceOver does not announce that you are on a new page. 

### 🕵️ Details

This was also able to be reproduced with VoiceOver on Safari

### What operating systems are you encountering the problem on?

macOS

### What browsers are you encountering the problem on?

Firefox

### What assistive technologies are you encountering the problem on?

VoiceOver

### 📸 Screenshot

https://github.com/user-attachments/assets/0f727205-6a4c-40db-8cc6-931106450413

**Comment by petertgiles** [comment]
@BrookeCameron can you reproduce this on Mac OS?
